app/views/teams/_form.html.erb

```diff
@@ -16,7 +16,9 @@
       <div class="flex flex-col gap-4 items-start justify-center w-full">
         <label class="text-sm">Upload team banner (< 1 MB)</label>
         <!-- From component: pass the hidden file input field with data attribute forms_target: "fileInput" for this to work  -->
-        <%= render "shared/components/file_select", hidden_file_field:  form.file_field(:banner, class: "hidden", data: { forms_target: "fileInput" }) %>
+        <%= render "shared/components/file_select",
+         hidden_file_field:  form.file_field(:banner, class: "hidden", data: { forms_target: "fileInput", file_type: "banner" }),
```

Comment:
File type passed again two times here. the parameter should have been preview. Is this even showing preview on selecting ?

Reply:
file type removed,Missed this part on the previous commit